code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -6,7 +6,7 @@
 	item_state = "laser"
 	fire_sound = 'sound/weapons/Laser.ogg'
 	slot_flags = SLOT_BELT|SLOT_BACK
-	w_class = 3
+	w_class = 4

```

Comment:
Oh. Weird. I guess I haven't tried shoving a shotgun into a bag in some time, then. Still, I don't think the advanced E-gun is as bulky as a shotgun.